src/operations/data.md

```diff
@@ -0,0 +1,37 @@
+# Data
+
+Toronto Community Network operates on principles of transparency, so most of our work is conducted in public digital spaces.
+The exception to this rule is when managing with personal information and sensitive information of our supporters and collaborators.
+This document describes our policy is finding the balance between working openly and restrictive access to certain information.
+
+## Document Management
+
+We store our documents on two main platforms:
+
+1. Our Toronto Mesh **GitHub** hosts our [main organizing repository](https://github.com/tomeshnet/toronto-community-network/) in addition other repositories for more specific content, like [data about our community network nodes](https://github.com/tomeshnet/node-list/).
+2. Our self-hosted **Nextcloud** drive contains two folders with different access levels:
+    - [tomeshnet](https://cloud.tomesh.net/f/480) stores semi-public information. Any Toronto Mesh member gets full access.
+    - [tomeshnet-trusted](https://cloud.tomesh.net/f/481) stores sensitive information, such as contact lists containing personal information and collaborator contracts that are explicitly asked to be kept private. Only active contributors to our working groups have access to this folder, and access shall be revoked when the member is no longer actively involved with the project. In special cases, access may be allowed to specific members with less history as contributor on an as-needed basis, by approval of the [Organizational and Program Governance Working Group](./working-groups.html#organizational-and-program-governance).
```

Comment:
We spoke about that sensitive information should be limited to those show have passed some sort of threshold of contribution first.  Such as "have been an active member for 1 year".  not just active working group members.

Reply:
Oops sorry, the one-year was in there and I tried to emphasize, but I see I have accidentally deleted it :D here is the revised. Good catch!

```suggestion
    - [tomeshnet-trusted](https://cloud.tomesh.net/f/481) stores sensitive information, such as contact lists containing personal information and collaborator contracts that are explicitly asked to be kept private. Only active members with more than one year of contribution in our working groups have access to this folder, and access shall be revoked when the member is no longer actively involved with the project. In special cases, access may be allowed to specific members with less history as contributor on an as-needed basis, by approval of the [Organizational and Program Governance Working Group](./working-groups.html#organizational-and-program-governance).
```